lib/radar/ical.rb

```diff
@@ -17,8 +17,8 @@ def parse_event(event)
       result = {
         id: event.start.to_s,
         url: event.url.to_s,
-        title: event.summary.force_encoding("utf-8"),
-        description: event.description.force_encoding("utf-8"),
+        title: (event.summary || "").force_encoding("utf-8"),
```

Comment:
Is it nil or undefined? (If nil, then `event.summary.to_s.force_encoding` would do the same, no?